pyscf/lib/libqcschema.py

```diff
@@ -0,0 +1,220 @@
+import json
+import numpy as np
+from pyscf.lib.parameters import BOHR
+
+def load_qcschema_json( file_name ):
+    # load qcschema output json file
+    data = None
+    with open(file_name,'r') as f:
+        data = json.load(f)
+    return data
+
+def load_qcschema_go_final_json( file_name ):
+    # load qcschema GO output json file 
+    # and return last 'trajectory' point's entries
+    # (this is the optimized molecule)
+    data = None
+    temp = None
+    with open(file_name,'r') as f:
+        temp = json.load(f)
+    data = temp["trajectory"][-1]
+    return data
+
+def combine_syms_coords(syms,coords,to_Angstrom = False, xyz=False):
+    # QCSchema molecules are split into symbols and coordinates,
+    #  combine them here.
+    # Optionally convert from Bohr to Angstrom.
+    # Returns: either a geometry string (default) or
+    #  optionally it can return an xyz format string.
+    if(to_Angstrom):
+        # convert Bohr to Angstrom
+        coords = coords*BOHR
+        
+    NAtoms = len(syms)
+    geo = np.reshape(coords, (NAtoms,3))
+
+    # Concatenate the symbols and coordinates along the second axis
+    combined = np.concatenate([syms[:, np.newaxis], geo], axis=1)
```

Comment:
just need `PySCF_atoms = list(zip(syms, geo))`. all remaining code are not needed. See https://pyscf.org/user/gto.html#geometry